proposals/umoci.md

```diff
@@ -0,0 +1,211 @@
+
+umoci has a fairly minimal feature set, and was intended from the outset to implement all of the key features which are needed from a reference implementation of the OCI Image Specifications.
+The help page for the latest version of `umoci` (`0.4.5` at the time of writing) is provided below:
+
+```
+NAME:
+   umoci - umoci modifies Open Container images
+
+USAGE:
+   umoci [global options] command [command options] [arguments...]
+
+VERSION:
+   0.4.5
+
+AUTHOR:
+   Aleksa Sarai <asarai@suse.com>
+
+COMMANDS:
+   raw      advanced internal image tooling
+   help, h  Shows a list of commands or help for one command
+
+   image:
+     config      modifies the image configuration of an OCI image
+     unpack      unpacks a reference into an OCI runtime bundle
+     repack      repacks an OCI runtime bundle into a reference
+     new         creates a blank tagged OCI image
+     tag         creates a new tag in an OCI image
+     remove, rm  removes a tag from an OCI image
+     stat        displays status information of an image manifest
+     insert      insert content into an OCI image
+
+   layout:
+     gc        garbage-collects an OCI image's blobs
+     init      create a new OCI layout
+     list, ls  lists the set of tags in an OCI layout
+
+GLOBAL OPTIONS:
+   --verbose      alias for --log=info
+   --log value    set the log level (debug, info, [warn], error, fatal) (default: "warn")
+   --help, -h     show help
+   --version, -v  print the version
+```
+
+For a more detailed explanation of umoci, see the [project website's guide][umoci-guide].
+
+[umoci]: https://umo.ci/
+[umoci-guide]: https://umo.ci/quick-start/
+
+## Proposal ##
+Change the ownership of the existing umoci project from openSUSE:
+
+  https://github.com/openSUSE/umoci
+
+And move it inside the `opencontainers` organisation:
+
+  https://github.com/opencontainers/umoci
+
+The import paths will correspondingly be "github.com/opencontainers/umoci" (umoci does have some Go API users, but since the project will be renamed -- and GitHub will add a redirect -- there will be no significant downstream impact of the change).
+In the future we may opt to use vanity imports (such as "umo.ci/cmd/umoci").
+
+The project's domain "umo.ci" will also be transferred to the Linux Foundation so that it can be managed by someone other than the maintainers (though maintainers must maintain the necessary administrative access to maintain the website).
+
+### Initial Maintainers ###
+Initial maintainers of the umoci project would be:
+
+* Aleksa Sarai <cyphar@cyphar.com> (@cyphar)
+* Tycho Andersen <tycho@tycho.ws> (@tych0)
+* Vincent Batts <vbatts@hashbangbash.com> (@vbatts)
+
+### Code of Conduct ###
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md
+
+### Governance and Releases ###
```

Comment:
I think this section is saying too much. The proposal here only needs to state high level what the governance model and release process is. This is going into too much detail that should be reserved for the actual repository. Also the current repository does not have this governance even represented. I would suggest mentioning it at a high level here and opening a PR in umoci that will get merged before or on moving of the repo.

Reply:
Sure, I am working on that now and I'd be happy to update this to be less specific.